docs/configuration.adoc

```diff
@@ -182,7 +182,7 @@ directory of this repository.
 
 Initialization scripts are shell (`.sh`) or Groovy (`.groovy`) scripts that are named using a convention that will ensure the order of their execution. For example, `00-start.sh` would be executed first and `99-finish.groovy` would be executed last.
 
-The builtin script that copies the files from the staging directory to the shared directory, database, or Oak repository is named `50-stage-type.groovy`. That way, any scripts that should be executed before files are deployed into the data space should be less than 50 and any scripts that should be executed after they are deployed should be greater than 50. Groovy scripts should not call `connect` unless they are greater than 50.
+The builtin script that copies the files from the staging directory to the shared directory or database is named `50-stage-type.groovy`. That way, any scripts that should be executed before files are deployed into the data space should be less than 50 and any scripts that should be executed after they are deployed should be greater than 50. Groovy scripts should not call `connect` unless they are greater than 50.
```

Comment:
Remove database. There is no database data space anymore. There's just a shared directory where the inetsoft.yaml file and a few other things are stored. The KV and blob stores are separate things.